### DISCUSSION TITLE
Unable to access minio console

### DISCUSSION THREAD

**Discussion by IgnisDa** [D1ceWard/minio_on_dokku#3] [General]
Hello. Thanks for the guide!

I am unable to access the minio console (on https://minio.diptesh.me:8443).

Here is the report:

http://sprunge.us/oiWf3W

I would really appreciate some help!

**Comment by d1ceward** [comment]
Hi,
What error did the web browser display when you did your tests?
Could it be a firewall problem?

**Reply by IgnisDa** [reply]
The web browser is stuck on the loading. 

>  Could it be a firewall problem?

Do I have to expose a specific port?

**Comment by IgnisDa** [comment]
Got it working with `ufw allow 8443`

Thanks for the hint!
